docs/start-deployment.md

```diff
@@ -107,6 +114,14 @@ To configure the proper domain name, you should change the setting and configure
 CL_SETTINGS_HOST=mywebsite.com # this could also be your external IP
 ```
 
+### Timezone
+
+Set the timezone for your application, a list of valid timezones can be found here.
+
+```bash
+CL_SETTINGS_CORE_TIMEZONE=Brussels
+```
```

Comment:
this is needed for the seeds to run, otherwise we get:

```bash
$ docker-compose run --rm web rake db:seed
Creating citizenlab-docker_web_run ... done
rake aborted!
ActiveRecord::RecordInvalid: Validation failed: Settings {:fragment=>"#/core/timezone", :error=>"Enum", :human_message=>"The property '#/core/timezone' value \"\" did not match one of the following values: Edinburgh, Lisbon, London, Monrovia, UTC, Amsterdam, Belgrade, Berlin, Bern, Bratislava, Brussels, Budapest, Casablanca, Copenhagen, Dublin, Ljubljana, Madrid, Paris, Prague, Rome, Sarajevo, Skopje, Stockholm, Vienna, Warsaw, Zagreb, Zurich, West Central Africa, Athens, Bucharest, Helsinki, Jerusalem, Kyiv, Riga, Sofia, Tallinn, Vilnius, Cairo, Harare, Kaliningrad, Pretoria, Baghdad, Istanbul, Kuwait, Minsk, Moscow, Nairobi, Riyadh, St. Petersburg, Tehran, Abu Dhabi, Baku, Muscat, Samara, Tbilisi, Volgograd, Yerevan, Kabul, Ekaterinburg, Islamabad, Karachi, Tashkent, Chennai, Kolkata, Mumbai, New Delhi, Sri Jayawardenepura, Kathmandu, Almaty, Astana, Dhaka, Urumqi, Rangoon, Bangkok, Hanoi, Jakarta, Krasnoyarsk, Novosibirsk, Beijing, Chongqing, Hong Kong, Irkutsk, Kuala Lumpur, Perth, Singapore, Taipei, Ulaanbaatar, Osaka, Sapporo, Seoul, Tokyo, Yakutsk, Adelaide, Darwin, Canberra, Hobart, Melbourne, Sydney, Brisbane, Guam, Port Moresby, Vladivostok, Magadan, New Caledonia, Solomon Is., Srednekolymsk, Auckland, Fiji, Wellington, Kamchatka, Marshall Is., Chatham Is., Samoa, Nuku'alofa, Tokelau Is., Azores, Cape Verde Is., Mid-Atlantic, Brasilia, Greenland, Buenos Aires, Montevideo, Newfoundland, Atlantic Time (Canada), Santiago, Caracas, Georgetown, La Paz, Puerto Rico, Eastern Time (US & Canada), Indiana (East), Bogota, Lima, Quito, Central Time (US & Canada), Guadalajara, Mexico City, Monterrey, Central America, Saskatchewan, Chihuahua, Mazatlan, Mountain Time (US & Canada), Arizona, Pacific Time (US & Canada), Tijuana, Alaska, Hawaii, American Samoa, Midway Island, International Date Line West in schema b5c24110-3e97-5fe2-ad8f-0a9c19efc5de"}
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/validations.rb:80:in `raise_validation_error'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/validations.rb:53:in `save!'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/transactions.rb:318:in `block in save!'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/transactions.rb:375:in `block in with_transaction_returning_status'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/connection_adapters/abstract/database_statements.rb:280:in `block in transaction'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/connection_adapters/abstract/transaction.rb:280:in `block in within_new_transaction'
/usr/local/bundle/gems/activesupport-6.0.3.5/lib/active_support/concurrency/load_interlock_aware_monitor.rb:26:in `block (2 levels) in synchronize'
/usr/local/bundle/gems/activesupport-6.0.3.5/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `handle_interrupt'
/usr/local/bundle/gems/activesupport-6.0.3.5/lib/active_support/concurrency/load_interlock_aware_monitor.rb:25:in `block in synchronize'
/usr/local/bundle/gems/activesupport-6.0.3.5/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `handle_interrupt'
/usr/local/bundle/gems/activesupport-6.0.3.5/lib/active_support/concurrency/load_interlock_aware_monitor.rb:21:in `synchronize'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/connection_adapters/abstract/transaction.rb:278:in `within_new_transaction'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/connection_adapters/abstract/database_statements.rb:280:in `transaction'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/transactions.rb:212:in `transaction'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/transactions.rb:366:in `with_transaction_returning_status'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/transactions.rb:318:in `save!'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/suppressor.rb:48:in `save!'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/persistence.rb:55:in `create!'
/cl2_back/db/seeds/citizenlab.rb:11:in `<top (required)>'
/usr/local/bundle/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/usr/local/bundle/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/cl2_back/db/seeds.rb:4:in `<top (required)>'
/usr/local/bundle/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/usr/local/bundle/gems/bootsnap-1.4.6/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:55:in `load'
/usr/local/bundle/gems/railties-6.0.3.5/lib/rails/engine.rb:559:in `load_seed'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/tasks/database_tasks.rb:440:in `load_seed'
/usr/local/bundle/gems/activerecord-6.0.3.5/lib/active_record/railties/databases.rake:331:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/rake-13.0.3/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:seed
(See full trace by running task with --trace)
ERROR: 1
```